airflow/providers/amazon/aws/operators/emr.py

```diff
@@ -16,18 +16,24 @@
 # specific language governing permissions and limitations
 # under the License.
 import ast
+import sys
 from typing import TYPE_CHECKING, Any, Dict, List, Optional, Sequence, Union
 from uuid import uuid4
 
-from airflow.compat.functools import cached_property
 from airflow.exceptions import AirflowException
 from airflow.models import BaseOperator
-from airflow.providers.amazon.aws.hooks.emr import EmrContainerHook, EmrHook
+from airflow.providers.amazon.aws.hooks.emr import EmrContainerHook, EmrHook, EmrServerlessHook
 from airflow.providers.amazon.aws.links.emr import EmrClusterLink
+from airflow.providers.amazon.aws.sensors.emr import EmrServerlessApplicationSensor, EmrServerlessJobSensor
 
 if TYPE_CHECKING:
     from airflow.utils.context import Context
 
+if sys.version_info >= (3, 8):
```

Comment:
I tihnk we are already fine with our util class (as in other providers) since we moved to 2.2+

Reply:
+1 You can import `cached_property` from [`airflow.compat.functools`](https://github.com/apache/airflow/blob/main/airflow/compat/functools.py).

Reply:
Sounds good, I'll make the change